pkg/sessions/cookie/session_store.go

```diff
@@ -129,6 +130,7 @@ func NewCookieSessionStore(opts *options.SessionOptions, cookieOpts *options.Coo
 // it into a slice of cookies which fit within the 4kb cookie limit indexing
 // the cookies from 0
 func splitCookie(c *http.Cookie) []*http.Cookie {
+	logger.Printf("WARNING: Sessions exceeds 4kb cookie limit. Multiple cookies are required for this session. Please use server side session storage (eg. Redis).")
```

Comment:
How do you feel about change `WARNING` to `DEPRECATED`?

Reply:
My thinking was that we haven't officially decided to deprecate yet, so shouldn't say it's deprecated? I was thinking we would add deprecated once we have gathered more feedback, though, changing it now, might make it easier to get feedback? 🤔